docker-compose.dev.yml

```diff
@@ -48,8 +48,6 @@ services:
             - ./docker/clickhouse/docker-entrypoint-initdb.d:/docker-entrypoint-initdb.d
             - ./docker/clickhouse/config.xml:/etc/clickhouse-server/config.xml
             - ./docker/clickhouse/users-dev.xml:/etc/clickhouse-server/users.xml
-        extra_hosts:
-            - 'host.docker.internal:host-gateway'
```

Comment:
Removed this as locally was failing by builds. Shouldn't have been pushed though.